airqo-platform/AirQo-frontend
# Ft device status summary frontend

What does this PR do?
Display device monitoring card Infos (Number of devices, Devices on solar, Devices on main power and number of devices due for maintenance)
Which JIRA card(s)?
The JIRA card for this PR is [here](https://airqoteam.atlassian.net/browse/NET-10)
How do I test out this PR?
Git clone this repo `cd AirQo-fronted/locate`, then run `npm install` to get all the necessary dependencies. After, run `npm run local` to view the changes. Remember to have the backend service running as well. Check the [API](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=1063367751) documentation for details.
Which endpoints should be /are ready for testing?
There are no endpoints for testing this PR.
